react/Header/Header.js

```diff
@@ -16,6 +16,18 @@ import employerLinkForLocale from './employerLinkForLocale';
 import StructuredDataSchema from './StructuredDataSchema/StructuredDataSchema';
 import { AUTHENTICATED, UNAUTHENTICATED, AUTH_PENDING } from '../private/authStatusTypes';
 
+const defaultNewBadgeTabForLocale = locale => (
+  locale === 'AU' ?
+    'Career Advice' :
+    'Profile'
```

Comment:
I'm just not sure wether we should not have taken off them all (from both AU and NZ). Is it a requirement?

Reply:
We're basically maintaining the existing behaviour in NZ for now.

Reply:
Yeah I get it, I just feel that it's a bit not logical, like it stopped being new for Australian people, but not for New Zealand people? Do they think slower than Australians? :/